rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,483 @@
+     - obs space:
+         name: msonet_winds_288
+         distribution:
+           name: "@DISTRIBUTION@"
+           halo size: 100e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: @OBSFILE@
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: MSONET_hofxs_winds_288.nc4
+             allow overwrite: true
+         io pool:
+           max pool size: 1
+         observed variables: [windEastward, windNorthward]
+         simulated variables: [windEastward, windNorthward]
+
+       obs operator:
+         name: Composite
+         components:
+         - name: VertInterp
+           hofx_scaling: true
+           hofx scaling field: wind_reduction_factor_at_10m
+           hofx scaling field group: GeoVaLs
```

Comment:
Do these hofx_scaling options require an external GSI geovals file, similar to the configurations you commented out below?

Reply:
No, JEDI will create the `wind_reduction_factor_at_10m` geovals itself. The comments below I will eventually delete. Those require special code in ufo to use which would override the JEDI created geovals including for f10m. I just wasn't ready to delete the commented out parts yet in case I needed to go back to my Phase 1 validation for something.

Reply:
That's weird. When I try to use wind_reduction_factor_at_10m, I get the error "field wind_reduction_factor_at_10m cannot be obtained from input fields." It seems like this variable is sourced from the background file?

Reply:
@spanNOAA, send me the path to your yaml and I will check it out.

Reply:
Here's the yaml file: /work/noaa/wrfruc/span/jedi_summer_retro/jedi_prepbufr_analysis/2023061012/3denvar_adpsfc_winds_287.yaml.
And here's the log file: /work/noaa/wrfruc/span/jedi_summer_retro/jedi_prepbufr_analysis/2023061012/fv3jedi_envar_conv.o2848743.

Reply:
I'm assuming that "cannot be obtained from input fields" means "cannot be derived from input fields"?

Reply:
Hmm that is strange. Yeah, I think it just comes from the input field `f10m`. I see you have that in your sfcdata so I wonder why it isn't working. We'll have to dig into that some more. 

p.s. don't take my current yamls to be a sane starting place, for now. I'm testing some changes related to the order of the obfilters and trying to remove most/all instances of `defer to post`. I'm also struggling some with the ObsErrorFactorPressureCheck (for winds).

Reply:
It seems that the issue was resolved by adding f10m to the state variable.

Reply:
Okay great! When I was looking at your yaml, `f10m` was in your state variables.

Reply:
I do notice that "defer to post" affects the results differently. I'm testing it out as well.